### PR TITLE
Feature/angular boilerplate

### PR DESCRIPTION
**Pull request by CloudNiner**
Sets up basic angular app with a single sample view.

Verified deployment build to `/dist` works as expected

### CODE REVIEW

**Comment by CloudNiner** [comment]
Also sets up Travis

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+indent_size = 4
+
+# We recommend you to keep these unchanged
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (flibbertigibbet) [inline .editorconfig:21]: This is just for Markdown files, right? I find it useful to have my editor trim whitespace for me.

File: Gruntfile.js [hunk 2]
```diff
@@ -0,0 +1,479 @@
...
+      options: {
+        assetsDirs: [
+          '<%= yeoman.dist %>',
+          '<%= yeoman.dist %>/images',
+          '<%= yeoman.dist %>/styles'
+        ]
+      }
+    },
+
+    // The following *-min tasks will produce minified files in the dist folder
```
Comment (flibbertigibbet) [inline Gruntfile.js]: Can we take out this commented block?

File: .editorconfig (hunk 1 above)
Comment (CloudNiner) [inline .editorconfig:21]: Yup. Default for all files above is true.

Either way this file came with the yo install, we could just remove it, I don't know anyone here who actually uses the editorconfig plugin. It's only installed by default on a few IDEs.

http://editorconfig.org/#download

File: Gruntfile.js (hunk 2 above)
Comment (CloudNiner) [inline Gruntfile.js]: Will do.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "grunt-google-cdn": "^0.4.3",
+    "grunt-karma": "*",
+    "grunt-newer": "^1.1.0",
+    "grunt-ng-annotate": "^0.9.2",
+    "grunt-svgmin": "^2.0.0",
+    "grunt-usemin": "^3.0.0",
+    "grunt-wiredep": "^2.0.0",
+    "jshint-stylish": "^1.0.0",
+    "karma-jasmine": "*",
+    "karma-phantomjs-launcher": "*",
```
Comment (flibbertigibbet) [inline package.json:31]: Why the wildcards for jasmine and PhantomJS launcher? Don't we want to require a minimum version?

**Comment by flibbertigibbet** [comment]
+1

File: package.json (hunk 3 above)
Comment (CloudNiner) [inline package.json:31]: Huh. So I went digging in the yo generator source to see if a commit message explains why they are wildcards, and those two dependencies are the only two not in the template package.json file. So I'm not sure where they came from, and I didn't want to spend more than a minute or two looking.

I think this is ok either way, it'll just pull the newest version. If we run into issues, we can always lock it down to a specific version later.

Comment (flibbertigibbet) [inline package.json:31]: Ok, shouldn't be a problem.

**Pull request merged**